src/Filters/HttpFilter.php

```diff
@@ -134,16 +136,19 @@ protected function addFiltersToQuery(Builder $builder)
     protected function filtersExact(Builder $query, $value, string $property): Builder
     {
         $property = self::sanitize($property);
-    
-        if ($query->getModel()->hasCast($property, ['date', 'datetime', 'immutable_date', 'immutable_datetime']) || $property === $query->getModel()->getCreatedAtColumn() || $property === $query->getModel()->getUpdatedAtColumn()) {
-            if (is_array($value) && (isset($value['start']) || isset($value['end']))) {
-                return $query->whereDate($property, '<=', $value['end'] ?? now()->addYears(50))
-                             ->whereDate($property, '>=', $value['start'] ?? now()->subYears(50));
-            }
+        
+        if ($query->getModel()->hasCast($property, ['date', 'datetime', 'immutable_date', 'immutable_datetime']
+            ) || $property === $query->getModel()->getCreatedAtColumn() || $property === $query->getModel()
+                                                                                               ->getUpdatedAtColumn()) {
+            $query->when($value['min'] ?? null, fn (Builder $query) => $query->whereDate($property, '>=', $value['min']));
```

Comment:
some miscommunication has happened. min max should only be used for numbers and start end for dates as it sounds more consistent with what we are checking.

Reply:
I'd use the same terminology for both. So either min/max, or start/end in both places.

Reply:
okay, now I see. Will revert the changes within daterange, and keep it only in numberrange.